pages/dkp/kommander/1.4/clusters/attach-cluster/cluster-with-network-restrictions/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: layout.pug
+navigationTitle: Attach a cluster with networking restrictions
+title: Attach a cluster with networking restrictions
+menuWeight: 10
+excerpt: How to attach an existing cluster that has additional networking restrictions
+beta: true
+---
+
+Use this option when you want to attach a cluster that is in a DMZ, behind a proxy server or a firewall, or that requires additional access information. This procedure gathers the information required to create a kubeconfig file for the network tunnel between Kommander and the cluster you want to attach.
```

Comment:
is it exactly the same file here ? Did you just move it ? This would help us reviewing it.

Reply:
@hectorj2f - same file, just some minor adjustments.

Had to rename the file so it was consistent, and so the link would work in the app.